module/easter_egg_func.py

```diff
@@ -44,3 +48,60 @@ def lei_che_ne_pensa_signorina(update: Update, context: CallbackContext) -> None
     check_log(update, context, "leiCheNePensaSignorina")
     message_text = EasterEgg.get_lei_che_ne_pensa_signorina()
     context.bot.sendMessage(chat_id=update.message.chat_id, text=message_text)
+
+
+def tomarchio_schedule(context: CallbackContext) -> None:
+    """Called once a day at 00:00 (ROME)
+    Schedules the "tomarchio_request" job that will run between 10:00 and 21:00 of the same day
+    Also, makes sure the file "data/soldini" exists
+
+    Args:
+        context (CallbackContext): context passed by the updater
+    """
+    if not path.exists("data/soldini"):
+        with open("data/soldini", 'w+') as soldini:
+            soldini.write("0")
+    #context.job_queue.run_once(tomarchio_request, when=time(hour=randint(10, 21), minute=randint(0, 59)))
```

Comment:
se non serve potremmo toglierlo

Reply:
In realtà serviva il rigo commentato e non quello sotto, che avevo usato solo per testare il tutto. Ho risolto